graviton/blackbox.py

```diff
@@ -725,7 +725,7 @@ def execute_one_dryrun(
             args = tuple(args)
 
             if method.returns.type != abi.Returns.VOID:
-                abi_return_type = method.returns.type
+                abi_return_type = cast(abi.ABIType, method.returns.type)
```

Comment:
I'm a bit leery about this new casts because of upcoming major refactoring in #45 (but they're not too big a deal to hold up upgrading to v2.0 of py-algorand-sdk)

Reply:
The only case of `method.returns.type` being `str` is `abi.Returns.VOID`, so I have to use the cast here, for mypy is not smart enough... Let me know how I can help with #45 after this merge in.